### PR TITLE
[KIECLOUD-217] - Kie data path standardization

### PR DESCRIPTION
**Pull request by ricardozanini**
Please see:
https://issues.jboss.org/browse/KIECLOUD-217

Signed-off-by: Ricardo Zanini <zanini@redhat.com>

### CODE REVIEW

File: modules/rhpam-apb/roles/deploy-businesscentral-monitoring/templates/businesscentral-monitoring-dc.yml.j2 [hunk 1]
```diff
@@ -200,7 +200,7 @@ spec:
               mountPath: "/etc/businesscentral-certs"
               readOnly: true
             - name: businesscentral-data
```
Comment (ricardozanini) [inline modules/rhpam-apb/roles/deploy-businesscentral-monitoring/templates/businesscentral-monitoring-dc.yml.j2]: This one needs to be renamed as well? @errantepiphany

Comment (spolti) [reply modules/rhpam-apb/roles/deploy-businesscentral-monitoring/templates/businesscentral-monitoring-dc.yml.j2]: would make sense move it to /opt/kie/certificates/{businesscentral,kieserver}

Comment (errantepiphany) [reply modules/rhpam-apb/roles/deploy-businesscentral-monitoring/templates/businesscentral-monitoring-dc.yml.j2]: Just checking in... I like /opt/kie/data/certs/* . Keeping everything in /opt/kie will be very helpful and organized.

Comment (ricardozanini) [reply modules/rhpam-apb/roles/deploy-businesscentral-monitoring/templates/businesscentral-monitoring-dc.yml.j2]: @spolti I was talking about the `businesscentral-data`, but you made a good point. The problem is, this mountPath should be changed in every template as well. Don't you think it's better to open a different JIRA (or PRs) to make this change?

Comment (ricardozanini) [reply modules/rhpam-apb/roles/deploy-businesscentral-monitoring/templates/businesscentral-monitoring-dc.yml.j2]: @errantepiphany ^

Comment (errantepiphany) [reply modules/rhpam-apb/roles/deploy-businesscentral-monitoring/templates/businesscentral-monitoring-dc.yml.j2]: @ricardozanini I am fine with the same jira, as long as it gets done across each of the images.

**Comment by ricardozanini** [comment]
@errantepiphany as per our internal conversation, the certificates path change has been dropped. I'm ok to merge this one.

**Pull request merged**